tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -260,6 +260,14 @@ def hasLength(length, error_func=None):
         else f"Value length {len(value)} does not match {length}.",
     )
 
+def hasLengthBetween(lower, upper, error_func=None):
+    """Validate that value (string or array) has a length matching length param."""
+    return make_validator(
+        lambda value: len(value) >= lower and len(value) <= upper,
+        lambda value: error_func(value, lower, upper)
+        if error_func
+        else f"Value length {len(value)} is not on the range [{lower}, {upper}].",
```

Comment:
@reitermb could I get some UX input on this please :)

Reply:
Could we adjust the grammar/style a little? 

`else f"Value length {len(value)} is not in the range [{lower} - {upper}].",` 

Changes being in vs on and - rather than ,